crates/storage/src/connection/class.rs

```diff
@@ -70,6 +70,28 @@ pub(super) fn insert_cairo_class(
     Ok(())
 }
 
+pub(super) fn insert_cairo_class_at(
```

Comment:
BTW, I think you won't need this if you're using `Connection.insert_state_update` to update the state (storage updates and class declarations). It also has the benefit that it's what the actual state update code in sync uses.